### PR TITLE
Use functions instead of strings in function nodes

### PR DESCRIPTION
**Pull request by breuleux**
This replaces the strings `"return_"`, `"universe_getitem"`, and so on that are currently in some constant nodes produced by the parser by builtin functions when appropriate, or functions in `myia.basics` when they are Myia-specific. Tests and representations are adjusted accordingly.

### CODE REVIEW

File: myia/basics.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        self.value = value
+
+
+def make_handle(object_type):
+    """Create a handle of the given type."""
+    return Handle(object_type)
+
+
+def global_universe_getitem(h):
+    """Set the value of a handle."""
```
Comment (notoraptor) [inline myia/basics.py]: I think it should be "Get the value of a handle" here.

File: myia/basics.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    return h.value
+
+
+def global_universe_setitem(h, value):
+    """Get the value of a handle."""
```
Comment (notoraptor) [inline myia/basics.py]: And "Set" here.

**Review by notoraptor — COMMENTED** [review]
I only checked changes in python backend, it seems good for me! But there is a typo in documentation for `global_universe` methods.

File: myia/basics.py (hunk 1 above)
Comment (breuleux) [reply myia/basics.py]: Oh, right, haha.

File: myia/ir/print.py [hunk 3]
```diff
@@ -131,7 +149,7 @@ def __init__(self):
         )
 
     def __call__(self, node):
-        if node.is_constant_graph():
+        if isinstance(node, Node) and node.is_constant_graph():
```
Comment (abergeron) [inline myia/ir/print.py:152]: This should always be called with a Node so that check should not necessary

**Review by abergeron — APPROVED** [review]

**Comment by abergeron** [comment]
I didn't really look at the html pages, but the rest looks good.

File: myia/ir/print.py (hunk 3 above)
Comment (breuleux) [reply myia/ir/print.py:152]: It is now used for graph_printer as well, which passes Graphs sometimes.

Comment (abergeron) [reply myia/ir/print.py:152]: Ok then.

**Pull request merged**